examples/webots/city_intersection/city_intersection.scenic

```diff
@@ -52,7 +52,7 @@ class LogImageAction(Action):
     def applyTo(self, obj, sim):
         print("Other Car Visible:", self.visible)
         
-        target_path = self.path + "/"
+        target_path = str(self.path) + "/"
```

Comment:
While we're changing this, we should probably make it work on Windows too by switching to `pathlib`. (I think the `os.path` and `shutil` functions accept path-like objects, so you might not even need to change the code below; using the `pathlib` equivalents would be fine too.)

Reply:
Good point, I'll make the change!